scripts/ci/lib.sh

```diff
@@ -1140,40 +1149,63 @@ post_process_test_results() {
         return 0
     fi
 
+    local slack_attachments_file="$1"
     local csv_output
     local extra_args=()
     local base_link
     local calculated_base_link
+    local create_jiras
+    local jira_project="ROX"
+    local prow_job_link
 
     set +u
     {
-        if is_in_PR_context || [[ "${PULL_BASE_REF:-unknown}" =~ ^release ]]; then
-            info "Converting JUNIT found in ${ARTIFACT_DIR} to CSV"
+        info "Post processing junit records to JIRA issues, BigQuery metrics and Slack attachments as appropriate"
+
+        prow_job_link="$(make_prow_job_link)"
+
+        if is_in_PR_context; then
+            if pr_has_label "ci-test-junit-processing"; then
+                create_jiras="true"
+            else
+                create_jiras="false"
```

Comment:
Thinking about this, perhaps we _should_ report failures related to cluster create/teardown problems, even for PRs? For those rare PRs which tinker with cluster creation/teardown logic (and are somewhat expected to cause a failure in that area), we could have a label that disables reporting? :thinking:

Reply:
Agreed. Something to add to the improvements epic.

Reply:
https://issues.redhat.com/browse/ROX-21405